lib/string.flow

```diff
@@ -1388,7 +1389,49 @@ toStringNoExtraQuotes(v: flow) -> string {
 }
 
 strRepeat(s : string, cnt : int) -> string {
-	if (cnt > 0) s + strRepeat(s, cnt - 1) else "";
+	if (cnt <= 0) {
+		"";
+	} else if (cnt == 1) {
+		s;
+	} else {
+		// list2string(generateList(0, cnt, \__ -> s));
+		p = generateProgressString(cnt, makeList1(s), 1, 1);
```

Comment:
@alstrup  like list2string(generateList(0, cnt, \__ -> s)) more . but maybe i didn't understand you